lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -524,11 +524,10 @@ private function getTablePrimaryKeys(Table $table)
      */
     private function getClassNameForTable($tableName)
     {
-        if (isset($this->classNamesForTables[$tableName])) {
-            return $this->namespace . $this->classNamesForTables[$tableName];
-        }
-
-        return $this->namespace . Inflector::classify(strtolower($tableName));
+        return $this->namespace . (
+            $this->classNamesForTables[$tableName] ??
```

Comment:
This ?? should also be at the beginning of next line.

Reply:
Should I indent?
```php
$this->classNamesForTables[$tableName]
    ?? Inflector::classify(strtolower($tableName))
```

Reply:
OF course, 4 spaces as usual. 👍